### PR TITLE
Implement fmt.Stringer for VolumeHandle

### PR DESCRIPTION
**Pull request by amdabhad**
- Implement fmt.Stringer for VolumeHandle which returns a volume handle instance in string format
- scale-core: https://github.ibm.com/IBMSpectrumScale/scale-core/issues/5355

### CODE REVIEW

**Review by Jainbrt — APPROVED** [review]

**Review by ste-roth — DISMISSED** [review]

**Review by sam6258 — CHANGES_REQUESTED** [review]
It would be better to implement the `fmt.Stringer` interface

**Comment by amdabhad** [comment]
> It would be better to implement the `fmt.Stringer` interface

Done

**Review by Jainbrt — APPROVED** [review]

File: driver/csiplugin/pkg/consistencygroup/consistencygroup.go [hunk 1]
```diff
@@ -117,3 +117,8 @@ func GetConsistencyGroupFilesetLinkPath(pvs *corev1.CSIPersistentVolumeSource) (
 
 	return cgFsetLinkPath, nil
 }
+
```
Comment (sam6258) [inline driver/csiplugin/pkg/consistencygroup/consistencygroup.go:120]: Add a compile time assertion for implementing the interface. 

```
var _ fmt.Stringer = VolumeHandle{}
```

Comment (amdabhad) [reply driver/csiplugin/pkg/consistencygroup/consistencygroup.go:120]: Done

**Review by sam6258 — APPROVED** [review]

**Pull request merged**